src/com/chschmid/jdotxt/gui/controls/JdotxtContentField.java

```diff
@@ -29,7 +29,7 @@
 import java.util.List;
 
 @SuppressWarnings("serial")
-public class JdotxtContentField extends JTextField{
+public class JdotxtContentField extends JTextArea {
```

Comment:
This does not play nice with scroll bar for some reason.  In some cases I see words which end at the boundary are not wrapped and not fully visible, and I could not find a way to view the obscured part of the text.  (In sinlge-line mode you just move cursor, and you will see the text, in multiline I cant find a way to access obscured text.)

![image](https://user-images.githubusercontent.com/8833179/93342868-83b8ba00-f838-11ea-8050-0cc4561ff0cf.png)

UPD: this behavior is observed in the "bad" state mentioned below, when maximized window is moved from one display to another display of a different size.